Sources/Logger/Loggers/SystemLogger.swift

```diff
@@ -11,21 +11,33 @@ import os
 /// Pre-built logger that wraps system os_logger
 public class SystemLogger: Logging {
     private let logEntryEncoder: LogEntryEncoding
+    private let prefix: String
     private var logger: OSLog
 
     public init(
         subsystem: String,
         category: String,
-        logEntryEncoder: LogEntryEncoding = LogEntryEncoder()
+        logEntryEncoder: LogEntryEncoding = LogEntryEncoder(),
+        prefix: String = ""
```

Comment:
Pojdme to udelat jinak, at to je fakt koser, kdyz uz ses to rozhodl testovat...

Injectuj tu funkci skrz konstruktor, defaultne ji nastav na ten `os_log` a v testu tam posli neco, co pak zvladnes assertovat. Vlastne tam jen posles closure s assertama, kde budes validovat to zavolani...